package.json

```diff
@@ -5,6 +5,10 @@
   "description": "React specific linting rules for ESLint",
   "main": "index.js",
   "scripts": {
+    "clean-built-types": "rm -f $(find . -maxdepth 1 -type f -name '*.d.ts*') $(find lib -type f -name '*.d.ts*' ! -name 'types.d.ts')",
+    "prebuild-types": "npm run clean-built-types",
+    "build-types": "tsc -p build.tsconfig.json",
+    "prepublishOnly": "npm run build-types",
```

Comment:
this should be done in prepack, since it's for inclusion in the tarball.

Reply:
I'll fix 👍 I will have to focus on a client project tomorrow so I will probably not have time to look at this PR again until Wednesday earliest

Reply:
This tab was open when I opened my computer this morning, so I squeezed it in anyhow 🙈 Most things should be fixed now